Makefile

```diff
@@ -135,6 +135,7 @@ test-env:
 	source test-env/bin/activate
 	pip3 install -r tests/requirements.txt
 
+CF_TEMPLATE_URL ?= https://s3.amazonaws.com/downloads.mesosphere.io/dcos-enterprise/testing/master/cloudformation/ee.single-master.cloudformation.json
```

Comment:
this is the strict CF template?

Reply:
This is permissive by default. The CI job in TeamCity (spark-build-PR-strict) has this set to a strict template.